dipy/io/tests/test_streamline.py

```diff
@@ -209,6 +209,15 @@ def test_low_io_vtk():
         npt.assert_array_almost_equal(tracks[1], streamline, decimal=4)
 
 
+@pytest.mark.skipif(not have_fury, reason="Requires FURY")
+def test_ply_io_vtk():
+    with InTemporaryDirectory():
+        fname = 'test.ply'
+
+        # Test save
+        save_vtk_streamlines(streamlines, fname)
+
```

Comment:
Can you complete the test? This can be made in a similar way as the other test cases in this module. Read the saved file and compare its length to the length of the streamlines object i.e., `tracks = load_vtk_streamlines(fname)` and `npt.assert_equal(len(tracks), len(streamlines))`. You could compare the content of both objects as follows: `npt.assert_array_almost_equal(tracks[1], streamline, decimal=4)`.